README.md

```diff
@@ -8,16 +8,17 @@
 | Plastic straws | Reusable metal straws |
 | Plastic cutlery | Metal cutlery set |
 | Throwing away fruits & vegetable discards | Composting discards & planting seeds |
-| Sealed nuts | Bulk nuts |
+| Packaged nuts | Bulk nuts |
 | Plastic juice bottles | Freshly squeezed juice |
 | Yogurt cups | Yogurt tubs served in a washable cup |
 | Coffee pods | Coffee beans and burr grinder |
 | Tea sachets | Loose-leaf tea and tea pot |
 | Plastic or paper soda cups | Reusable and washable mugs, bottles, or cups |
 | Plastic cutting boards | Wooden cutting boards | 
-| Paper or wax baking liners | Sillicon liners | 
+| Paper or wax baking liners | Silicone liners | 
 | Plastic cooking utensils | Wooden cooking utensils | 
 | Paper nakpins | Washable cloth napkins |
+| Disposable Ziploc bags | Washable silicone zip storage bags
```

Comment:
Please create a different PR for this change since this is a new proposal and not a simple typo fix like the rest of your changes :trollface: